### DISCUSSION TITLE
Ability to type in max-width

### DISCUSSION THREAD

**Discussion by HusseinT2000** [notaapp/talk#308] [Feature Requests]
I would find it great if it was possible to put in your own value for max width.

I tend to have nota open on the side in a narrow window so being able to have a lower max width than 720 would be great especially when viewing tables in markdown mode.

**Comment by astoilkov** [comment]
Quick question. Do you want the padding to be smaller? If we allow setting your own max width that won't make the padding smaller (a.k.a. have more space when the window is smaller).
